bokehjs/src/coffee/widget/slider.coffee

```diff
@@ -16,6 +17,7 @@ class SliderView extends ContinuumView
     html = @template(@model.attributes)
     @$el.html(html)
     @render()
+    dom_util.add_css_classes(@$el, @model.css_classes())
```

Comment:
This line should be in `render()`, otherwise `change` event will remove custom classes (see line 14).